3001-4000/LIT3871Miracle.xml

```diff
@@ -6,7 +6,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
          <titleStmt>
             <title xml:id="t1">Miracle of Mary: How a woman consigned to the devil the son she conceived on Easter eve</title>
```

Comment:
the Devil? The title 1 is the most apt, can the names of the lady be introduced there?

Reply:
I collected the titles from the various sources, so that people can hopefully find the miracle they're looking for. The name is present in t2